test/unit_tests/braket/ahs/test_driving_field.py

```diff
@@ -107,6 +107,111 @@ def test_discretize():
     assert discretized_field.detuning == detuning_mock.discretize.return_value
 
 
+def test_from_lists():
+    times = [0, 0.1, 0.2]
+    amplitudes = [0.5, 0.8, 0.9]
+    detunings = [0.3, 0.7, 0.6]
+    phases = [0.2, 0.4, 0.6]
+
+    dr_field = DrivingField.from_lists(times, amplitudes, detunings, phases)
+    assert dr_field.amplitude.time_series.values() == amplitudes
+    assert dr_field.detuning.time_series.values() == detunings
+    assert dr_field.phase.time_series.values() == phases
+
+    assert dr_field.amplitude.time_series.times() == times
+    assert dr_field.detuning.time_series.times() == times
+    assert dr_field.phase.time_series.times() == times
+
+
+def test_concatenate():
+    dr_field_1 = DrivingField.from_lists(
+        times=[0, 0.1, 0.2],
```

Comment:
I would suggest you structure this test the way you did the one below, so you're testing the whole lists and not just the first/last elements.

Reply:
Agree, updated